app/javascript/packs/personal-key-page-controller.js

```diff
@@ -69,16 +24,20 @@ function downloadForIE(event) {
   window.navigator.msSaveBlob(blob, filename);
 }
 
+function trackAcknowledgment(clickEvent) {
+  if (clickEvent.target.checked) {
+    trackEvent('IdV: personal key acknowledged');
+  } else {
+    trackEvent('IdV: personal key un-acknowledged');
+  }
```

Comment:
For these sorts of behaviors, I think it's more conventional to have a single event and include the toggle state as a property of the event. For what we'd discussed last week about common click behaviors, I'm [thinking about](https://github.com/18F/identity-idp/pull/7109#issuecomment-1271598814) how to handle a checkbox like this in #7109, and wondering if "checked" could be a suitable, consistent property name.

What would you think about... ?

```suggestion
  trackEvent(
    'IdV: personal key acknowledgment toggled',
    { checked: clickEvent.target.checked },
  );
```